content/en/hub/scale/dev-notes.md

```diff
@@ -143,9 +143,42 @@ midclt call -job kubernetes.update '{"pool": "pool_name_here"}'
 
 This will setup kubernetes on the defined pool. For the first time, it may take a few minutes for the k8s cluster to properly initialise itself. Moving on, if you have a pool configured for kubernetes, kubernetes will start automatically on boot.
 
-**Using Kubernetes**
+**Deploying Kubernetes Workloads**
 
-SCALE does not support deploying workloads to kubernetes cluster officially yet, however users interested in using k8s can do so manually via shell. SCALE has helm3 pre-installed, so users can leverage helm to deploy their workloads.
+SCALE has API support to deploy kubernetes workloads. SCALE manages kubernetes workloads with integration of Helm v3 and using a concept of catalogs ( For information on catalogs please refer to https://github.com/truenas/charts ).
+
+Each catalog item points to an application which will be deployed to kubernetes. This application is an enhanced helm chart which deploys the application to TrueNAS SCALE kubernetes cluster.
+
+Right now we have a catalog item `ix-chart` (https://github.com/truenas/charts/tree/master/test/ix-chart/2010.0.1) which has the ability to deploy single docker image workloads. Please refer to the chart link to see more details about what features / configuration are supported.
+
+An example to deploy plex docker image would be:
+```
+midclt call -job chart.release.create '{"catalog": "OFFICIAL", "train": "test", "item": "ix-chart", "values": {"image": {"repository": "plexinc/pms-docker"}, "portForwardingList": [{"containerPort": 32400, "nodePort": 32400}], "volumes": [{"datasetName": "transcode", "mountPath": "/transcode"}, {"datasetName": "config", "mountPath": "/config"}, {"datasetName": "data", "mountPath": "/data"}], "workloadType": "Deployment", "gpuConfiguration": {"nvidia.com/gpu": 1}}, "version": "2010.0.1", "release_name": "plex"}'
+```
+
+To update the chart release, an example would be:
+```
+midclt call -job chart.release.update 'plex' '{"values": {"portForwardingList": [{"containerPort": 32400, "nodePort": 32401}]}}'
+```
+
+To delete the chart release, an example would be:
+```
+midclt call -job chart.release.delete plex
+```
+
+To upgrade the chart version, an example would be ( assuming we have 2010.0.2 version ):
+```
+midclt call -job chart.release.upgrade 'plex' '{"item_version": "2010.0.2"}'
+```
+
+To rollback the chart release to previous chart version, an example would be:
+```
+midclt call -job chart.release.rollback 'plex' '{"item_version": "2010.0.1"}'
+```
+
+**Using Kubernetes via CLI**
+
+SCALE does not support workloads created manually via kubectl / helm or direct interaction with kubernetes api. For users interested in using `kubectl` / `helm` to retrieve information can do so directly.
```

Comment:
I'm not sure what the second sentence is trying to say?
```suggestion
SCALE does not support workloads created manually via kubectl / helm or direct interaction with kubernetes api. For users interested in using `kubectl` / `helm` to retrieve information can do so directly.
```

Reply:
The idea is to let the reader know that users can use `kubectl` / `helm` commands to retrieve resources information directly and then below it on how they will be able to use the commands.
Although we don't recommend doing this as middleware retrieves what we consider useful information but we would like users to play with them in case they are developing catalog items etc.